src/app/saved-jobs/page.tsx

```diff
@@ -1,10 +1,69 @@
+'use client';
 import Link from 'next/link';
+import { ScrollArea } from '../../components/ui/scroll-area';
+import SavedJob from './components/SavedJob';
+import ButtonWithBadge from './components/ButtonWithBadge';
+import { useState } from 'react';
+
+import type { Job } from './placeHolderData';
+import { jobsArr, jobsAmounts } from './placeHolderData';
 
 export default function SavedJobsPage() {
+  const [currentTab, setCurrentTab] = useState<string>('Saved');
+  const filteredArray: Array<Job> = jobsArr.filter(
+    (job) => job.savedLocation == currentTab,
```

Comment:
We should use `===` here as well. There's a few other places that you use `==`, I'm not going to comment on all of them but you should change them all to `===`